apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -227,22 +227,21 @@ class FileManager extends Plugin {
   */
   async writeMultipleFiles(filePaths, fileData, folderPath) {
     try {
-      let alert = true
-      for (let i = 0; i < filePaths.length; i++) {
-        const installPath = folderPath + "/" + filePaths[i]
+      filePaths.forEach(async (filePath, index) => {
+        const installPath = folderPath + "/" + filePath
+        const alert = !index
```

Comment:
You have two variables named the same thing only differentiated by scope which isn't ideal. Your code should communicate intent as much as possible and if your have an inner alert and an outer one then what is the purpose of the outer one.